marked-element.html

```diff
@@ -154,6 +154,13 @@
       this._attached = false;
     },
 
+    /**
+     * Unindents the markdown source that will be rendered.
+     */
+    unindent: function(text) {
+      this._unindent(text);
```

Comment:
No need. It would not be possible to change the signature without it becoming a breaking change. This would be a non-problem if there was some true notion of `protected` in JS, but there isn't.